angular/projects/admin/src/app/admin/document-type/document-type-form/document-type-form.component.html

```diff
@@ -88,57 +89,98 @@
   </mat-card>
 
   <!-- A dynamic list of form fields that can be added or removed -->
-  <mat-card class="document-type-static-card">
-    <mat-card-header>
-      <mat-card-title>Conent type fields</mat-card-title>
-    </mat-card-header>
-    <mat-card-content>
+  <section class="document-type-form-fields">
+    <div class="header">
+      <h4 class="mat-body-1">Content type fields</h4>
+      <div class="actions">
+        <button mat-raised-button
+                color="primary"
+                class="mat-elevation-z0"
+                (click)="addField(null, null, true)">Add Field</button>
+      </div>
+    </div>
+    <div>
       <mat-radio-group formControlName="indexTitle">
         <div formArrayName="fields">
           <div class="row"
                *ngFor="let field of fieldForms.controls; let i=index"
-               [formGroupName]="i">
-            <div id="radio-isTitle">
-              <mat-radio-button [value]="i" matTooltip="Set this field as title"></mat-radio-button>
-            </div>
-            <div class="col">
-              <mat-form-field class="full-width">
-                <input matInput
-                       placeholder="Key"
-                       formControlName="key">
-              </mat-form-field>
-            </div>
-            <div class="col">
-              <mat-form-field>
-                <input matInput
-                       placeholder="Title"
-                       formControlName="title">
-              </mat-form-field>
-            </div>
-            <div class="col">
-              <mat-form-field>
-                <mat-select placeholder="Type"
-                            formControlName="type">
-
-                  <mat-option *ngFor="let fieldType of fieldTypes"
-                              [value]="fieldType.type">
-                    {{fieldType.title }}
-                  </mat-option>
-                </mat-select>
-              </mat-form-field>
-            </div>
-            <div class="col">
-              <button mat-icon-button
-                      (click)="deleteField(i)">
-                <mat-icon>delete</mat-icon>
-              </button>
-            </div>
+               [formGroupName]="i"
+               id="ct-field-{{i}}">
+            <mat-card class="document-type-static-card field-card">
+              <mat-card-content class="field-card-content">
+                <div id="radio-isTitle">
```

Comment:
I think it'd be good to add a label for the radio buttons or something that clarify what it does. Right now it's just a hovering tooltip. 

Perhaps a label such as _"Use as title"_ or _"is title"_

Reply:
<img width="1239" alt="Screen Shot 2019-05-20 at 11 35 53" src="https://user-images.githubusercontent.com/15650438/57995202-77610780-7af3-11e9-8824-800c526ccd46.png">
The UI looks bad if we have a label for title field. What do you think? @DennisAlund

Reply:
Let's go with this solution for now